src/test/java/org/mockitousage/junitrule/MockitoJUnitTestRuleTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2017 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockitousage.junitrule;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoTestRule;
+
+public class MockitoJUnitTestRuleTest {
+
+    @Rule
+    public MockitoTestRule mockitoRule = MockitoJUnit.rule(this);
+
+    // Fixes #1578: Protect against multiple execution.
```

Comment:
Is this Mockito ticket?

Reply:
This was taken from `MockitoJUnitRuleTest` and it felt like a useful comment for future reference.